ui/modules/settings-user.js

```diff
@@ -31,6 +31,17 @@ module.exports = function(core, config, store) {
 		});
 	});
 
+	core.on("user-dn", function(user, next) {
+		if (JSON.stringify(user.old) !== JSON.stringify(user.user)) {
```

Comment:
Why do we have this if condition? It will almost never be equal, and JSON.stringify is a slow, recursive function.